panel-app/src/components/app-root/app-root.css

```diff
@@ -0,0 +1,35 @@
+#top-bar {
```

Comment:
I like using SCSS instead of CSS to avoid repeating selectors. For example: 

```scss
 #top-bar {
	height: 60px;
	margin: 0;
	background: linear-gradient(270deg, #451c5c -1.34%, #333357 64.73%, #333357 87.2%);

	.header-container {
		padding: 16px 0;
		margin: 0 0 0 1.5rem;
		display: flex;
		align-items: center;

		#coveo-logo-img {
			height: 24px;
			width: 94px;
		}
	}

	#header-separator {
		border-radius: 4px;
		background: #ee8279;
		width: 2px;
		height: 22px;
		margin: 0 11px;
	}

	.top-bar-text {
		color: #fff;
		font-family: Gibson;
		font-size: 14px;
		font-style: normal;
		font-weight: 300;
		line-height: 21px; /* 150% */
		letter-spacing: 0.042px;
	}
}
```

Reply:
We want want to do that conversion into another PR too. 
I can give you some quick tips on SCSS if you aren't familiar with it.

Reply:
True. Makes much sense to use Scss. I'll add this conversion to a new PR